pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -325,10 +295,6 @@ def __init__(self, *args, **kargs):
         self.opts = {
             'connect': 'auto', # defaults to 'all', unless overridden to 'finite' for log-scaling
             'skipFiniteCheck': False, 
-            'fftMode': False,
-            'logMode': [False, False],
-            'derivativeMode': False,
-            'phasemapMode': False,
```

Comment:
These opts might be something users depend on? I hope not. Deprecating these would be tricky.